tests/test_all_rules.py

```diff
@@ -632,7 +632,8 @@ def test_integration_tag(self):
                     # checks if an index pattern exists if the package integration tag exists
                     integration_string = "|".join(indices)
                     if not re.search(rule_integration, integration_string):
-                        if rule_integration == "windows" and re.search("winlog", integration_string):
+                        if rule_integration == "windows" and re.search("winlog", integration_string) or \
+                                rule_integration in definitions.ANALYTIC_PACKAGES:
```

Comment:
So none of these will be tagged?

Reply:
@Mikaayenson the rules will be tagged, for example with `lmd` but are not required to have a matching index pattern defined in the rule. This is because the data is typically reliant on, for say, `endpoint` (`log-endpoint.events*`. But we need the `lmd` tag to build the `related_integrations` field and to do field validation with the schemas.